pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -949,7 +949,11 @@ class Build {
 
             try {
                 context.timeout(time: buildTimeouts.NODE_CHECKOUT_TIMEOUT, unit: "HOURS") {
-                    repoHandler.checkoutAdopt()
+                    if (useAdoptShellScripts) {
+                        repoHandler.checkoutAdopt()
+                    } else {
+                        context.checkout context.scm
```

Comment:
@M-Davies This seems to checkout 2 different repos depending on if the useAdoptShellScripts flag is enabled. checkoutAdopt will clone openjdk-build and checkout scm will clone ci-jenkins-pipeline. Unless I've misconfigured something on my side but I don't think this is what is expected.

Reply:
I think that's me getting confused on what exact repo we're checking out. I intend to address any issues as part of https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/issues/34